src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -251,12 +251,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             If Not PropertyControlData.IsSpecialValue(value) Then
 
-                Dim uIntValue As UInteger = 0
```

Comment:
This was originally because OutputType was returning a string in CPS and when it was empty, the whole page would be broken instead of a single property having wrong value. However if we change outputtype to intproperty we can probably remove this as well if DTE returns an int.